lib/new_relixir/instrumenters/external.ex

```diff
@@ -0,0 +1,30 @@
+defmodule NewRelixir.Instrumenters.External do
+  @moduledoc """
+  This instrumenter measures total time of each external request
+  made in the context of an application request.
```

Comment:
Since we're not providing any built-in adapters yet, I suppose users will have to roll out their own. I was thinking that maybe we can add some documentation in the README file explaining how to do that, with examples using popular HTTP client libraries. Also, I think it's a good idea to document the format of metadata we're handling here: `%{url: binary}`.

Reply:
I just stumbled over this, and yes, I have no immediate clue how to integrate this with, for example, HTTPoison. Some clues on this would be very helpful.